Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/MainActivity.java

```diff
@@ -9,71 +8,79 @@
 import android.database.Cursor;
 import android.os.Bundle;
 import android.preference.PreferenceManager;
-import android.support.v7.widget.GridLayoutManager;
-import android.support.v7.widget.RecyclerView;
+import android.support.annotation.Nullable;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
 import android.util.Log;
 import android.view.Menu;
 import android.view.MenuInflater;
 import android.view.MenuItem;
 import android.view.View;
-import android.widget.ImageButton;
-import android.widget.Toolbar;
 
 import org.dolphinemu.dolphinemu.NativeLibrary;
 import org.dolphinemu.dolphinemu.R;
-import org.dolphinemu.dolphinemu.adapters.GameAdapter;
+import org.dolphinemu.dolphinemu.adapters.PlatformPagerAdapter;
+import org.dolphinemu.dolphinemu.fragments.PlatformGamesFragment;
+import org.dolphinemu.dolphinemu.model.Game;
 import org.dolphinemu.dolphinemu.model.GameDatabase;
 import org.dolphinemu.dolphinemu.model.GameProvider;
 import org.dolphinemu.dolphinemu.services.AssetCopyService;
 
 /**
- * The main Activity of the Lollipop style UI. Shows a grid of games on tablets & landscape phones,
- * shows a list of games on portrait phones.
+ * The main Activity of the Lollipop style UI. Manages several PlatformGamesFragments, which
+ * individually display a grid of available games for each Fragment, in a tabbed layout.
  */
-public final class GameGridActivity extends Activity implements LoaderManager.LoaderCallbacks<Cursor>
+public final class MainActivity extends AppCompatActivity implements LoaderManager.LoaderCallbacks<Cursor>
 {
 	private static final int REQUEST_ADD_DIRECTORY = 1;
 
-	private static final int LOADER_ID_GAMES = 1;
-	// TODO When each platform has its own tab, there should be a LOADER_ID for each platform.
+	/**
+	 * It is important to keep track of loader ID separately from platform ID (see Game.java)
+	 * because we could potentially have Loaders that load things other than Games.
+	 */
+	public static final int LOADER_ID_ALL = 100; // TODO
```

Comment:
It's just a reminder that LOADER_ID_ALL doesn't actually do anything and will need to be implemented.